docs/data/charts/axis/GroupedYAxes.tsx

```diff
@@ -47,7 +47,6 @@ const salesData = [150, 120, 200, 180, 90, 160, 140, 110, 85];
 const profitData = [45, 35, 80, 65, 25, 55, 50, 40, 30];
 
 const chartConfig = {
-  width: 600,
```

Comment:
This was breaking the page on mobile.

Before:

<img width="413" height="697" alt="Image" src="https://github.com/user-attachments/assets/9eeebe08-1fb5-48c5-a8a7-5a52a444d9a9" />

After:

<img width="417" height="719" alt="Image" src="https://github.com/user-attachments/assets/717c6d4a-7b63-4933-9161-c1900afa8f68" />